kill.go

```diff
@@ -60,8 +61,12 @@ func parseSignal(rawSignal string) (syscall.Signal, error) {
 	if err == nil {
 		return syscall.Signal(s), nil
 	}
-	signal, ok := signalMap[strings.TrimPrefix(strings.ToUpper(rawSignal), "SIG")]
-	if !ok {
+	sig := strings.ToUpper(rawSignal)
+	if !strings.HasPrefix(sig, "SIG") {
+		sig = "SIG" + sig
+	}
+	signal := unix.SignalNum(sig)
+	if signal == 0 {
```

Comment:
sig 0 is valid to check if a process is alive

Reply:
Dropping the knowledge bombs :smile:

Looks like an oversight in the `golang.org/x/sys/unix` library's `SignalNum` library: (https://godoc.org/golang.org/x/sys/unix#SignalNum)

> SignalNum returns the syscall.Signal for signal named s, or 0 if a signal with such name is not found. The signal name should start with "SIG".

Given that there's not a name for "SIG zero" and that we catch the explicit "numeric input" case further up, this should be reasonable, right?

Reply:
reasonable I think until someone adds the equivalent of a new SIGNONE or SIGPING text name option to the list of kill signals and someone adds it to golang unix.SignalNum() :-0

Reply:
maybe a comment line or two explaining why the name lookup should fail for zero for now even though the raw 0 option should not fail..

Reply:
> reasonable I think until someone adds the equivalent of a new SIGNONE or SIGPING text name option to the list of kill signals and someone adds it to golang unix.SignalNum()

@tklauser fyi (discussion above)

Reply:
> sig 0 is valid to check if a process is alive

but it does not have a name, so...

@mikebrow I'm not sure if adding any comments to the code will improve it. It's pretty clear anyway -- we treat it as a number first, and pass on any valid number to Syscall.Signal(), letting it return an error if the number is wrong.

If it's not a number, we treat it as a name and use `unix.SignalNum` to resolve. The doc for the function clearly states that it returns `0` if a signal with such name is not found, so we check for `0`. 

All that is very clear from the code.

Reply:
> > reasonable I think until someone adds the equivalent of a new SIGNONE or SIGPING text name option to the list of kill signals and someone adds it to golang unix.SignalNum()
> 
> @tklauser fyi (discussion above)

That's indeed an oversight in `x/sys/unix`. The better option would have been to return `-1` in case no signal with the given name is found. But I think, we probably cannot change that now as we would break existing users :(

Reply:
ok, i see. this logic should be fine since it's a name lookup